genkit-tools/common/src/eval/localFileDatasetStore.ts

```diff
@@ -90,6 +90,7 @@ export class LocalFileDatasetStore implements DatasetStore {
       datasetId: id,
       schema,
       targetAction,
+      metrics: req.metrics,
```

Comment:
nit: Would be nice to indicate that these are names of metrics evaluators that will be used for evaluation. For anything named metrics, i usually expect to see numbers in it. Maybe thats just me haha. But how about metricsForEval or metricNames?

Reply:
Good point. Made it `metricRefs`